### DISCUSSION TITLE
chatgpt can not generate

### DISCUSSION THREAD

**Discussion by srmdnadem** [GorvGoyl/ChatGPT-Writer#14] [Report Bugs and Issues]
There is an issue with your application. It still needs to be buffered with the generation of text. Please fix it. I use google chrome

**Comment by GorvGoyl** [comment]
it's fixed now. please update it to v 2.0 from chrome store and let me know if you are still facing the issue

**Reply by srmdnadem** [reply]
Ok, I will do that, and thank you for doing that

**Comment by anewbies** [comment]
Se estiver usando o Google Chrome, entre na configuração chrome://settings/syncSetup e marque a caixa on "verificação de ortografia aprimorada" e o problema estará resolvido.
![image](https://user-images.githubusercontent.com/14927641/213434388-b31201f0-4ac8-4fcc-a0eb-626e83ff2ef3.png)
